vector/src/main/res/layout/adapter_item_vector_unknown_devices_header.xml

```diff
@@ -23,6 +23,7 @@
         <ImageView
             android:id="@+id/heading_image"
             android:layout_width="24dp"
+            android:tint="?attr/vctr_general_icon_tint_color"

```

Comment:
Using `?vctr_list_header_primary_text_color` is sufficient, and the textColor attribute of the TextView above should also be set to `?vctr_list_header_primary_text_color`, and the root RelativeLayout background should be set to `?attr/vctr_list_header_background_color` instead of `?android:attr/colorBackground`